src/App/router/router.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { createBrowserRouter } from 'react-router-dom';
+import { MainPage, RepoPage, ErrorPage } from 'App/pages';
+import { loader as mainPageLoader } from 'App/pages/MainPage/loader/loader';
+import { loader as repoPageLoader } from 'App/pages/RepoPage/loader/loader';
+import Root from './Root';
+
+export const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <Root />,
+    errorElement: <ErrorPage />,
+    children: [
+      {
+        index: true,
+        errorElement: <ErrorPage />,
+        path: ':org',
+        loader: mainPageLoader,
+        element: <MainPage />,
+      },
+      {
+        path: ':org/:repo',
```

Comment:
не особо гибко получается... а что если ты захочешь ещё страницу добавить? лучше сделать что-то типо
```suggestion
        path: ':org/repo/:repo',
```